modules/bridge/src/main/ts/ephox/bridge/components/dialog/Dropzone.ts

```diff
@@ -1,20 +1,32 @@
-import { FieldSchema, StructureSchema } from '@ephox/boulder';
-import type { Result } from '@ephox/katamari';
+import { FieldSchema, StructureSchema, ValueType } from '@ephox/boulder';
+import type { Optional, Result } from '@ephox/katamari';
 
 import { type FormComponentWithLabel, type FormComponentWithLabelSpec, formComponentWithLabelFields } from './FormComponent';
 
 export interface DropZoneSpec extends FormComponentWithLabelSpec {
   type: 'dropzone';
   context?: string;
+  dropAreaLabel?: string;
+  buttonLabel?: string;
+  allowedFileTypes?: string;
+  allowedFileExtensions?: string[];
```

Comment:
This is public API, so please make sure the JIRA documentation notes mentions these new properties

And they should probably go in a new changelog entry as well